decidim-accountability/app/forms/decidim/accountability/admin/result_form.rb

```diff
@@ -7,12 +7,11 @@ module Admin
       class ResultForm < Decidim::Form
         include TranslatableAttributes
         include TranslationsHelper
+        include HasTaxonomyFormAttributes
```

Comment:
```suggestion
        include Decidim::HasTaxonomyFormAttributes
```

Reply:
ok, but, shouldn't we do that for all includes then?

Reply:
yeah ... let's leave it for the moment